go/vt/sqlparser/ast.go

```diff
@@ -3000,18 +3000,25 @@ func (node *ValuesFuncExpr) replace(from, to Expr) bool {
 // SubstrExpr represents a call to SubstrExpr(column, value_expression) or SubstrExpr(column, value_expression,value_expression)
```

Comment:
Nit: `column`->`column/val`

Reply:
I thought about it, but if I reuse `ColName` , e.g.:

```
SUBSTR openb STRING ',' value_expression closeb
  {
    $$ = &SubstrExpr{Name: &ColName{Name: NewColIdent($3)}, From: $5, To: nil}
  }
```

How could I figure out later if it's a static string or column name? How could I stop `walkSubtree` in this case?
Maybe instead of having extra `StrVal *SQLVal`, I'll add some boolean member (static/resolved/...) to mark it's a static string expression.

Reply:
Oh. You misunderstood :). I just wanted the `SubstrExpr` comment to be updated to say that the input can be a column or a value, to reflect the code change.

Reply:
Ok, how about now - take a look.